modules/gcp-stackdriver-export/variables.tf

```diff
@@ -21,3 +21,15 @@ variable "exports" {
   # Example: {"exclude-all-foo" = "logName=\"projects/__var.project_id__/logs/foo\"}
   default = {}
 }
+
+variable "exported_logs_storage_class" {
+  default = "REGIONAL"
+}
+
+variable "exported_logs_storage_region" {
+  default = "europe-north1-a"
```

Comment:
Is `europe-north1-a` something we want here for a default?

Reply:
It's the default zone in other Exekube models, so I think it's best to remain consistent. We override this on the gpii-infra end, of course, like any Exekube user.

Reply:
Ok it makes sense, I saw the US one in our modules and that's why asked. While I agree it's not great to have zones hardcoded, it's not bothering me at this time or anything. I guess at some point in future it would be interesting to try to start our stuff in eu region :) (it's a bit closer after all).